manifests/manifest.FIREFOX.json

```diff
@@ -0,0 +1,15 @@
+{
+   "background": {
+		"scripts": ["build/background.js"]
+	},
+   "browser_specific_settings": {
+		"gecko": {
+         "id": "{6bc19aa2-7d91-4791-b5d4-8d84b5f3cdad}",
```

Comment:
I'm not exactly sure what this UUID actually means, but setting a unique one like this does allow the build to pass 🤔